docs/static/ds4/docs.min.js

```diff
@@ -26,32 +26,23 @@ $_mod.def("/makeup-screenreader-trap$0.1.0/util",function(n,t,e,r,o){"use strict
 $_mod.def("/makeup-modal$0.1.0/index",function(e,t,n,a,r){"use strict";function d(){if(i){p.untrap(i),o.untrap(i);var e=document.createEvent("Event");e.initEvent("unmodal",!1,!0),i.dispatchEvent(e),i=null}return i}function u(e){d(),i=e,p.trap(i),o.trap(i);var t=document.createEvent("Event");return t.initEvent("modal",!1,!0),i.dispatchEvent(t),i}var i,o=e("/makeup-keyboard-trap$0.2.0/index"),p=e("/makeup-screenreader-trap$0.1.0/index");n.exports={modal:u,unmodal:d}});
 $_mod.def("/@ebay/skin$7.4.4/docs/_babel/base/transition",function(e,n,t,i,r){"use strict";function o(e){for(var n=window.getComputedStyle(e).transitionDuration.replace(s,"")?1:0,t=e.firstElementChild;t;)n+=o(t),t=t.nextElementSibling;return n}function a(e){var n,t;return window.requestAnimationFrame?(n=requestAnimationFrame(function(){n=requestAnimationFrame(e)}),t=cancelAnimationFrame):(n=setTimeout(e,26),t=clearTimeout),function(){n&&(t(n),n=void 0)}}var m="transitionend",s=/0m?s(?:, )?/g;t.exports=function(e,n,t){function i(){s||(s=!0,e.removeEventListener(m,r,!0),f?(f(),d.remove(v)):d.remove(n))}function r(){++c===u&&(s=!0,e.removeEventListener(m,r,!0),d.remove(n),t&&t())}var s,u,c=0,d=e.classList,v=n+"-init",f=a(function(){e.addEventListener(m,r,!0),d.add(n),d.remove(v),u=o(e),f=void 0,0===u&&i()});return d.add(v),i}});
-$_mod.installed("@ebay/skin$7.4.4","makeup-roving-tabindex","0.2.3");
-$_mod.main("/makeup-roving-tabindex$0.2.3","");
-$_mod.installed("makeup-roving-tabindex$0.2.3","custom-event-polyfill","1.0.7");
-$_mod.installed("makeup-roving-tabindex$0.2.3","nodelist-foreach-polyfill","1.2.0");
-$_mod.installed("makeup-roving-tabindex$0.2.3","makeup-navigation-emitter","0.2.2");
-$_mod.main("/makeup-navigation-emitter$0.2.2","");
-$_mod.installed("makeup-navigation-emitter$0.2.2","custom-event-polyfill","1.0.7");
-$_mod.installed("makeup-navigation-emitter$0.2.2","nodelist-foreach-polyfill","1.2.0");
-$_mod.installed("makeup-navigation-emitter$0.2.2","makeup-key-emitter","0.1.0");
+$_mod.installed("@ebay/skin$7.4.4","makeup-roving-tabindex","0.3.0");
+$_mod.main("/makeup-roving-tabindex$0.3.0","");
```

Comment:
@agliga Can you just rerun `yarn` in your local branch again then do `yarn build` to check you have the correct js. Wasn;t expecting these to change.

Reply:
Reran, seems to be fine now